homeassistant/components/mobile_app/webhook.py

```diff
@@ -426,6 +429,12 @@ async def webhook_update_sensor_states(hass, config_entry, data):
 
         unique_id = sensor[ATTR_SENSOR_UNIQUE_ID]
 
+        # Work around bug in iOS app https://github.com/home-assistant/iOS/issues/599
+        # It's sending state but not registering sensor
+        if unique_id == "connectivity_sim_2":
+            resp[unique_id] = {"success": True}
+            continue
```

Comment:
I wonder if this issue is a result of a None value during registration (Suggest fix in #36566).
In that case, this should not be an issue. If this can happen in general and is ok to mute like this, shouldn't it be better to consider another logging level instead?

Reply:
Yeah you're right, let's drop this part.